content/aips/AAVE_V3_ETH_AAVE_RISK_PARAMS.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Update AAVE V3 ETH Risk Parameters
+author: Marc Zeller (@marczeller), Aave-Chan initiative
+shortDescription: Update AAVE V3 ETH Risk Parameters
+discussions: https://governance.aave.com/t/arfc-align-aave-risk-parameters-on-aave-v3-ethereum-market-with-aave-v2/12656
+created: 2023-04-16
+---
+
+## Simple Summary
+
+This AIP proposal presents Aave with the opportunity to update AAVE risk parameters on the Aave V3 Ethereum Pool
+
+## Motivation
+
+This AIP proposes the alignment of the AAVE risk parameters on the Aave V3 Ethereum market with those on Aave V2, as the current parameters on Aave V3 are less attractive for users considering migration.
+
+By matching the Aave V2 parameters, we aim to encourage migration and ensure a smoother transition for users. This proposal does not suggest any changes to Aave V3 caps or interest rate strategies.
+
+To make Aave V3 more attractive for migration and to match Aave V2 parameters, we propose the following changes:
+
+- Loan-to-Value (LTV): Increase the LTV from 60% to 66%.
+- Liquidation Threshold (LT): Increase the LT from 70% to 73%.
+- Liquidation Penalty (LP): Maintain the current LP of 7.5%, as it is already in alignment with Aave V2.
+
+No changes to Aave V3 caps or interest rate strategies are proposed in this AIP.
```

Comment:
```suggestion
```

No listings either?, would just remove, doesn't really make sense to list what the proposal doesn't do. Especially as already written above.